src/components/Messages.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+class Messages extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      messages: {},
+      newMess: "",
+      newAuthor: ""
+    };
+
+    this.quoteChange = this.quoteChange.bind(this);
+    this.authorChange = this.authorChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    fetch(`/api/messages`)
+      .then(response => {
+        return response.json();
+      })
+      .then(data => {
+        this.setState({
+          messages: data
+        });
+      });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    const result = {
+      message: this.state.newMess,
+      person: this.state.newAuthor
+    };
+    console.log(result);
+    if (
+      this.state.newMess.length > 0 &&
```

Comment:
I suspect the entire condition can be rewritten as `if(this.state.newMess.length && this.state.newAuthor.length)`. If it's a string it will have the length property and we can take advantage of the fact that if length is greater than zero, it will be truthy